plasma_framework/test/src/exits/payment/outputGuardHandlers/PaymentOutputGuardHandler.test.js

```diff
@@ -0,0 +1,52 @@
+const PaymentOutputGuardHandler = artifacts.require('PaymentOutputGuardHandlerWrapper');
+
+const { expectRevert } = require('openzeppelin-test-helpers');
+const { expect } = require('chai');
+
+const { addressToOutputGuard } = require('../../../../helpers/utils.js');
+
+
+contract('PaymentOutputGuardHandler', ([alice]) => {
+    const TEST_OUTPUT_TYPE = 1;
+
+    beforeEach('setup contracts', async () => {
+        this.handler = await PaymentOutputGuardHandler.new(TEST_OUTPUT_TYPE);
+    });
+
+    describe('isValid', () => {
+        it('should fail when preimage is not empty', async () => {
+            const nonEmptyPreimage = '0x11';
+            const guard = addressToOutputGuard(alice);
+
+            await expectRevert(
+                this.handler.isValid(guard, TEST_OUTPUT_TYPE, nonEmptyPreimage),
+                'Pre-imgage of the output guard should be empty',
+            );
+        });
+
+        it('should fail when output type mismatch', async () => {
+            const preimage = '0x';
```

Comment:
is it necessary to explicitly put `'0x'` in this case? cannot it be just `''` as empty preimage?

Reply:
truffle would not translate `string` from js to `bytes` in solidity, however, it translates `hex` to `bytes`. That's why our tests uses hex to represent bytes input.

Reply:
so what about extracting a const such as  `EMPTY_BYTES='0x'` ?